docs/README.md

```diff
@@ -0,0 +1,15 @@
+# Lyra Docs
+
+This website is built using [Docusaurus 2](https://v2.docusaurus.io/), a modern static website generator.
+
+### Local Development
+
+First build `@stellar/lyra-api` (a dependency of this project) by cd'ing to `@stellar/lyra-api` and running `yarn build`.
+
+Then, come back to this directory and run:
+
+```
+$ yarn start
+```
+
```

Comment:
just created a ticket to figure out how to automatically build needed subrepos on `yarn start`, but for now we can manually build it

Reply:
Yeah development experience is getting a little rough. Should spend some time on that soon